### DISCUSSION TITLE
Support "releaseTimestamp" field for Docker datasource

### DISCUSSION THREAD

**Discussion by zharinov** [renovatebot/renovate#24901] [Suggest an Idea]
**What would you like Renovate to be able to do?**
We need to make one more request for each tag retrieved from Docker Hub API.
Though, each manifest contains lots of history records.

**Describe the solution you'd like**
Choose latest timestamp and save it in cache. 
The invalidation strategy remains open, because tags are mutable.

**Describe alternatives you've considered**
Omit "releaseTimestamps" for Docker datasource.

**Additional context**
Related to #4689.
Also, see this thread: https://github.com/renovatebot/renovate/pull/5760#discussion_r396059874

**Comment by rarkins** [comment]
The challenge here is:
- Queries required is O(n) number of tags
- Tag release dates are not immutable, so we have to choose between potentially outdated cache vs many queries

Right now the releaseTimestamp is used only for the final selected tag/update in order to determine if it's stable or not. So potentially we could "lazily" query it (perhaps another datasource API), however in future we might possibly do *decision making* based on release timestamp and therefore want to consider multiple tags at a time.

**Comment by viceice** [comment]
Does the docket registry API last-modified-since header oder something similar? So we can do a head request to check if there is a change, so we need to update our cache. 🤔

**Comment by zharinov** [comment]
Good point, I'll check this tomorrow

**Comment by HonkingGoose** [comment]
@zharinov Did you manage to check this comment from @viceice?

> Does the docket registry API last-modified-since header oder something similar? So we can do a head request to check if there is a change, so we need to update our cache. 🤔

I don't know if we can move this issue from`status:requirements` to `status:ready` now, or if we need to think some more on this?

Or maybe we no longer want or need this feature, in which case we can close it.
If we still want this feature, can you select a appropriate priority level?

**Comment by viceice** [comment]
This is required for `stabillityDays` to work.

Added for searchabillity
